CameraOverlay/tests/orchestrator_test.py

```diff
@@ -0,0 +1,38 @@
+"""Test Script For Orchestrator Class"""
```

Comment:
Don't we already have this in `master`??

Reply:
Yes we do. I have not updated my branch

Reply:
Please update your branch with latest master!